### ISSUE TITLE
Notifications not received

### ISSUE THREAD

**Issue by giohappy** [geosolutions-it/smb-ios#10]
I'm not receiving any notification on the iOS app. 
Indeed I can't see my iphone device registered on the backend. Why?

**Comment by pragmaware** [comment]
Yes, see the geosolutions-it/smb-portal#206  issue about it. The firebase registration call fails.

**Comment by giohappy** [comment]
What I don't understand is why you're using the same registration-id. See my questions in my comment there https://github.com/geosolutions-it/smb-portal/issues/206#issuecomment-533064305 @pragmaware

**Comment by pragmaware** [comment]
The notifications about the tracks being validated are now working.
The prize, badge and bike notifications should be working but I don't know how to test them.
